admin/javascript/LeftAndMain.BatchActions.js

```diff
@@ -289,6 +290,7 @@
 					type: 'POST',
 					data: this.serializeArray(),
 					complete: function(xmlhttp, status) {
+						$('.cms-content-fields').removeClass('loading');
```

Comment:
Can you reference `jQuery` over `$`, to keep consistency with the rest of this code?